games-emulation/fbalpha_libretro/fbalpha_libretro-0.2.97.42~git.recipe

```diff
@@ -9,8 +9,8 @@ REVISION="1"
 srcGitRev="099c24b166f08b671f703051de1daffb2e6d5eb7"
 SOURCE_URI="https://github.com/libretro/fbalpha/archive/$srcGitRev.tar.gz"
 CHECKSUM_SHA256="2ac4a4cf35aaf3380195b4804d9574be33ae52f4f95b7bd3ab8be450de5c8673"
-SOURCE_FILENAME="fbalpha-$portVersion~git.tar.gz"
-SOURCE_DIR="fbalpha-$srcGitRev"
+SOURCE_FILENAME="fbalpha-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="fbalpha-$portVersion-$srcGitRev"
```

Comment:
Oops, SOURCE_DIR was correct before your last update.
```diff
-SOURCE_DIR="fbalpha-$portVersion-$srcGitRev"
+SOURCE_DIR="fbalpha-$srcGitRev"
```
(As you know, SOURCE_DIR is the name of the top directory of the tarball, and the tarball we're using here is a snapshot. So no `$portVersion` here. Just the `$srcGitRev` :)

Reply:
Daaaaaamn ok ok fixing that whoops